README.md

```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
 Open Projects 2021
+
+
+![socialdistancing_mask_detection](https://user-images.githubusercontent.com/88554453/128603773-dceafea5-7f4f-46a8-8303-98dc23235368.jpg)
+
+# Abstract 
+
+The social distance checker is an opencv project in which we monitor the distances between the people and if the distance comes out to be less than the threshold value (calibrated six feet in terms of pixels) then we indicate the violation by surrounding them with a red bounding box and with the help of AI/ML/pretrained files , if the person does not wear a mask then his face gets surrounded by a red bounding box.
+
+# Motivation
+
+The Outbreak of COVID-19 has affected almost everyone across the globe. During this pandemic, people are 
+advised not to be in close contact with others to reduce the spread of the disease, but due to the negligence 
+of many, we continue to face the repercussions of this pandemic
+As the name suggests, social distancing implies that people should physically distance themselves from 
+one another, reducing the spread of contagious diseases like COVID. So, in order to monitor if people are 
+maintaining social distance or not, we are working on the project named, Social distance checker. 
+
+# Application
```

Comment:
Include a tabulated cost structure depicting the price of any paid software used before the Applications. For eg: https://github.com/marsiitr/IoT-ChatBot#cost-structure